{"name": "# Enter the main page as a new user and verify Подтверждение доступа is here", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.TimeoutException: Message: \nStacktrace:\nBacktrace:\n\tOrdinal0 [0x01163733+2504499]\n\tOrdinal0 [0x010FC401+2081793]\n\tOrdinal0 [0x01002628+1058344]\n\tOrdinal0 [0x0102CEBB+1232571]\n\tOrdinal0 [0x010568F2+1403122]\n\tOrdinal0 [0x01045EBA+1334970]\n\tOrdinal0 [0x01054CDB+1395931]\n\tOrdinal0 [0x01045D4B+1334603]\n\tOrdinal0 [0x010222B4+1188532]\n\tOrdinal0 [0x01023149+1192265]\n\tGetHandleVerifier [0x012DFB8C+1512252]\n\tGetHandleVerifier [0x0138B0DF+2214031]\n\tGetHandleVerifier [0x011E4BC3+484211]\n\tGetHandleVerifier [0x011E3E69+480793]\n\tOrdinal0 [0x0110218D+2105741]\n\tOrdinal0 [0x011066E8+2123496]\n\tOrdinal0 [0x01106827+2123815]\n\tOrdinal0 [0x0110FB73+2161523]\n\tBaseThreadInitThunk [0x772BFA29+25]\n\tRtlGetAppContainerNamedObjectPath [0x77497A9E+286]\n\tRtlGetAppContainerNamedObjectPath [0x77497A6E+238]\n\n", "trace": "  File \"c:\\users\\rapid\\appdata\\local\\programs\\python\\python39\\lib\\site-packages\\behave\\model.py\", line 1329, in run\n    match.run(runner.context)\n  File \"c:\\users\\rapid\\appdata\\local\\programs\\python\\python39\\lib\\site-packages\\behave\\matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n  File \"features\\steps\\004_register_and_enter.py\", line 8, in rgstr_as_nw_usr\n    context.app.main_page.rgstr_as_nw_usr()\n  File \"C:\\Everything\\IT\\Testing\\Automation_08_09_2019\\kit_rodionov\\pages\\main_page.py\", line 119, in rgstr_as_nw_usr\n    actual_txt = wait.until(EC.presence_of_element_located((TXT_HERE))).text\n  File \"c:\\users\\rapid\\appdata\\local\\programs\\python\\python39\\lib\\site-packages\\selenium\\webdriver\\support\\wait.py\", line 87, in until\n    raise TimeoutException(message, screen, stacktrace)\n"}, "steps": [{"name": "Given Loginpage", "status": "passed", "start": 1624397664411, "stop": 1624397666694}, {"name": "Then Click on Ekaterinburg/Moscow", "status": "passed", "start": 1624397666694, "stop": 1624397667370}, {"name": "And Register as a new user", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.TimeoutException: Message: \nStacktrace:\nBacktrace:\n\tOrdinal0 [0x01163733+2504499]\n\tOrdinal0 [0x010FC401+2081793]\n\tOrdinal0 [0x01002628+1058344]\n\tOrdinal0 [0x0102CEBB+1232571]\n\tOrdinal0 [0x010568F2+1403122]\n\tOrdinal0 [0x01045EBA+1334970]\n\tOrdinal0 [0x01054CDB+1395931]\n\tOrdinal0 [0x01045D4B+1334603]\n\tOrdinal0 [0x010222B4+1188532]\n\tOrdinal0 [0x01023149+1192265]\n\tGetHandleVerifier [0x012DFB8C+1512252]\n\tGetHandleVerifier [0x0138B0DF+2214031]\n\tGetHandleVerifier [0x011E4BC3+484211]\n\tGetHandleVerifier [0x011E3E69+480793]\n\tOrdinal0 [0x0110218D+2105741]\n\tOrdinal0 [0x011066E8+2123496]\n\tOrdinal0 [0x01106827+2123815]\n\tOrdinal0 [0x0110FB73+2161523]\n\tBaseThreadInitThunk [0x772BFA29+25]\n\tRtlGetAppContainerNamedObjectPath [0x77497A9E+286]\n\tRtlGetAppContainerNamedObjectPath [0x77497A6E+238]\n\n", "trace": "  File \"c:\\users\\rapid\\appdata\\local\\programs\\python\\python39\\lib\\site-packages\\behave\\model.py\", line 1329, in run\n    match.run(runner.context)\n  File \"c:\\users\\rapid\\appdata\\local\\programs\\python\\python39\\lib\\site-packages\\behave\\matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n  File \"features\\steps\\004_register_and_enter.py\", line 8, in rgstr_as_nw_usr\n    context.app.main_page.rgstr_as_nw_usr()\n  File \"C:\\Everything\\IT\\Testing\\Automation_08_09_2019\\kit_rodionov\\pages\\main_page.py\", line 119, in rgstr_as_nw_usr\n    actual_txt = wait.until(EC.presence_of_element_located((TXT_HERE))).text\n  File \"c:\\users\\rapid\\appdata\\local\\programs\\python\\python39\\lib\\site-packages\\selenium\\webdriver\\support\\wait.py\", line 87, in until\n    raise TimeoutException(message, screen, stacktrace)\n"}, "start": 1624397667370, "stop": 1624397684740}, {"name": "Then Verify text: \"Подтверждение доступа\" is here", "status": "skipped", "start": 1624397687303, "stop": 1624397687303}], "start": 1624397663225, "stop": 1624397687303, "uuid": "6308d915-4ca3-416b-8d61-731c3ce05e7f", "historyId": "6763c87e95498097c4ce3c0a1d92382c", "fullName": "# Enter the main page as a new user and verify Подтверждение доступа is here: # Enter the main page as a new user and verify Подтверждение доступа is here", "labels": [{"name": "severity", "value": "normal"}, {"name": "feature", "value": "# Enter the main page as a new user and verify Подтверждение доступа is here"}, {"name": "framework", "value": "behave"}, {"name": "language", "value": "cpython3"}]}